### ISSUE TITLE
Link colors in card content grids

### ISSUE THREAD

**Issue by kevincrafts** [CuBoulder/tiamat-theme#776]
Since the card content grid always has a white background, the link colors should always be blue.

However, it looks like if you put a card content grid into a section with a black content background, the links are set to gold:

<img width="1527" alt="image" src="https://github.com/CuBoulder/tiamat-theme/assets/349713/04430087-1a1d-4b7f-a8c9-adcc9f6ca531">

https://live-ucbdev-sandbox-wendy.pantheonsite.io/bachelor-science-aerospace-engineering
